atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaTest.java

```diff
@@ -76,6 +77,28 @@ public void testRenderJava8OptionalsWhenRequested() throws IOException {
                         containsString("import java.util.Optional")));
     }
 
+    @Test
+    public void testIgnoreTableNameLengthFlag() throws IOException {
+        Schema schema = new Schema("Table", TEST_PACKAGE, Namespace.EMPTY_NAMESPACE);
+        schema.ignoreTableNameLengthChecks();
+        String longTableName = String.join("", Collections.nCopies(100, "x"));
+        schema.addTableDefinition(longTableName, getSimpleTableDefinition(TABLE_REF));
```

Comment:
This looks odd as the ``TABLE_REF`` is not related to ``longTableName``at all, although I can see that this tests the functionality here as javaTableName only affects the index definitions. We can skip the javaTableName from the definition as it is optional or pass in a TableRef created from longTableNames.